src/chapters/razor-views/template-conditionals.rst

```diff
@@ -0,0 +1,171 @@
+.. How do we conditionally display data?
+
+Conditionals in a Template
+==========================
+
+In addition to iteration, Razor can also add or remove content on a
+webpage based on certain conditions. Going back to the coffee example, we could
+generate the ordered list ONLY IF ``coffeeOptions`` contains data. If the
+list of coffee types is empty, then there is no need to include the ``<ol>`` 
+element. Instead, the template could include a ``<p>`` element with text stating 
+that there are no options to select.
+
+Display Content ``if/else``
+---------------------------
+
+To include conditional logic to display a Razor template element, we use a basic C# ``if`` clause.
+
+The general syntax for including a conditional in Razor is:
+
+.. sourcecode:: guess
+
+   @if (condition)
+   {
+      // HTML element(s) to add or additional template logic
+   }
+
+Above, ``condition`` represents any expression that can be evaluated to ``true`` or ``false`` 
+(ie, a boolean).
+
+If ``condition`` evaluates to ``true``, then Razor adds the HTML element to
+the webpage and the content is displayed in the view. If ``condition`` is
+``false``, then Razor does NOT generate the element, and the content stays
+off the page.
+
+.. admonition:: Example
+
+   Assume that ``userSelection`` represents a string property on ``ViewBag``.
+
+   .. sourcecode:: guess
+      :linenos:
+
+      @if (ViewBag.userSelection == "Instant")
+      {
+         <p>Are you sure about that?</p>
+      } 
+      else if (ViewBag.userSelection == "French Roast")
+      {
+         <p>Oooh la la!</p>
+      } 
+      else
+      {
+         <p>Thanks for your order</p>
+      }
+
+The ``@if`` statement in line 1 compares a user's brew choice to the string
+``"Instant"``. If they match, then Razor adds the paragraph element to the
+view. If they do not match, then the condition on line 5 is evaluated. If that 
+``else if`` statement is true, then a paragraph element with different internal
+text is added. Lastly, if neither of those conditions is met, then the paragraph
+element with text on line 11 is rendered. 
+
+.. admonition:: Note
+
+   We don't add an additional ``@`` symbols on lines 5 and 9 in the 
+   example above. ``if``, ``else if``, and ``else`` clauses are dependent
+   and treated as one C# code block in Razor.
```

Comment:
I love this note! I always mess this up and add too many at signs and end up with build errors.